packages/xo-web/src/common/intl/messages.js

```diff
@@ -1606,6 +1606,7 @@ const messages = {
     'Restore {nVms, number} VM{nVms, plural, one {} other {s}} from {nVms, plural, one {its} other {their}} {oldestOrLatest} backup.',
   oldest: 'oldest',
   latest: 'latest',
+  restoreVmBackupsMacAddress: 'Generate new mac addresses',
```

Comment:
`generateNewMacAddresses`? I don't think we have to namespace the IDs if it's not necessary. This allows us to reuse messages as much as possible.